src/PrestaShopBundle/Resources/views/Admin/Multistore/dropdown.html.twig

```diff
@@ -43,7 +43,7 @@
               {% for shop in group.shops %}
                 <li class="multistore-dropdown-shop">
                   <a class="multistore-dropdown-shop-name{% if shop.hasMainUrl() == false %} multistore-dropdown-no-url">{{ shop.name }}</a>{% else %}" href="{{ app.request.requestURI ~ '&setShopContext=s-' ~ shop.id }}">{{ shop.name }}</a>{% endif %}
-                  {% if shopCustomizationChecker.isConfigurationCustomizedForThisShop(configurationKey, shop) == true %}
+                  {% if shopCustomizationChecker.isConfigurationCustomizedForThisShop(configurationKey, shop, isGroupShopContext) == true %}
```

Comment:
I don't see where the variable `isGroupShopContext` is defined. I looked into this file and MultistoreController

Reply:
it's line 146 in the `MultistoreController`